html/includes/common/availability-map.inc.php

```diff
@@ -12,41 +12,58 @@
  * the source code distribution for details.
  */
 
+$sql = dbFetchRow('SELECT `settings` FROM `users_widgets` WHERE `user_id` = ? AND `widget_id` = ?', array($_SESSION["user_id"], '1'));
+$widget_mode = json_decode($sql['settings']);
```

Comment:
We should probably pass true to json_decode here to keep in line with how it's done elsewhere.